pkg/token/token_test.go

```diff
@@ -152,7 +160,7 @@ func TestSTSEndpoints(t *testing.T) {
 }
 
 func TestVerifyTokenPreSTSValidations(t *testing.T) {
-	b := make([]byte, maxTokenLenBytes+1, maxTokenLenBytes+1)
```

Comment:
Just curious, what's the reason for removing capacity?

Reply:
the IDE told me to? it was redundant

Reply:
i can get rid of it for now